icu4c/source/common/uloc_keytype.cpp

```diff
@@ -222,6 +222,11 @@ initFromResourceBundle(UErrorCode& sts) {
                 if (U_FAILURE(sts)) {
                     break;
                 }
+                // TODO: For now we skip loading entries mapping to metazone short ids,
+                // which have 4 characters.
+                if (isTZ && uBcpTypeId.length() == 4) {
+                    continue;
+                }
```

Comment:
Should there be a followon issue noted here? (ICU known issue)

Reply:
It is actually primarily a CLDR issue, filed https://unicode-org.atlassian.net/browse/CLDR-15023